client/src/api/profiles.js

```diff
@@ -1,3 +1,11 @@
-export const getProfiles = () => {
-  return fetch("/api/profiles").then(res => res.json());
+export const getProfiles = profile => {
```

Comment:
```suggestion
export const postProfile = profile => {
```

The name `getProfiles` is a bit misleading because the function does not get profiles, it actually posts a profile to be added to the database. Therefore we can name it `postProfile` instead which is closer to what the function does.

Naming might not seem that important because code will work whatever the name is, but since code is mostly read by other developers it then becomes quite important that naming is done well so other developers can quickly understand what's happening!